packages/dashboard-frontend/src/services/workspace-client/devworkspace/devWorkspaceClient.ts

```diff
@@ -80,7 +117,9 @@ export class DevWorkspaceClient extends WorkspaceClient {
   private readonly showAlert: (alert: AlertItem) => void;
 
   constructor(@inject(KeycloakSetupService) keycloakSetupService: KeycloakSetupService,
-    @inject(AppAlerts) appAlerts: AppAlerts) {
+    @inject(AppAlerts) appAlerts: AppAlerts,
+    @multiInject(IDevWorkspaceEditorProcess) private editorProcesses: IDevWorkspaceEditorProcess[],
```

Comment:
@benoitf Could you please clarify why did you use @multiInject for IDevWorkspaceEditorProcess?

Because I can find only
```
container.bind(IDevWorkspaceEditorProcess).to(DevWorkspaceEditorProcessTheia).inSingletonScope();
```
https://github.com/eclipse-che/che-dashboard/pull/376/files#diff-fb91aea76f5a40d3573cac09e17def0370d0dee56e1365729095996afa56b62eR34

Reply:
Hello @olexii4 
I use multi-inject to not tied the logic to theia directly but to 'editor processing'

As I want to plug other libraries for other editors (like VS Code / intellij) I've used multi-inject (and each editor process being responsible on saying if it's applying or not the stuff) vs having all editors being injected directly in that class